pkg/cmd/roachtest/tests/cluster_to_cluster.go

```diff
@@ -207,7 +207,7 @@ SELECT
         fnv64(crdb_internal.trim_tenant_prefix(key),
               substring(value from 5))
     ) AS fingerprint
-FROM crdb_internal.scan(crdb_internal.tenant_span($1))
+FROM crdb_internal.scan(crdb_internal.tenant_span($1::INT))
```

Comment:
We should probably plumb the dst.name in and use that instead considering we're moving everything tenant related to be addressed by name, and generally change to using the new fingerprint method. In case you're feeling up to it, otherwise I can send out a patch separately.

Reply:
I say let's address the immediate flake for now. I assume this whole function will get replaced once your fingerprint job is ready for large scale testing.

Reply:
The advantage of having methods here that don't use the name is that if they don't need to look up the name then the function itself is a pretty clean utility function that is just constructing bytes rather than one that can fail.

Reply:
>  this whole function will get replaced once your fingerprint job is ready for large scale testing

This switch can be made now, i'll send out a patch